python/doc/examples/meta_modeling/general_purpose_metamodels/plot_taylor_approximation.py

```diff
@@ -97,15 +97,15 @@
 # %%
 # Let :math:`\vect{x}_0 \in \Rset^2` be a reference point where
 # the quadratic approximation is evaluated.
-# The second order Taylor expansion is:
+# The second order Taylor expansion of the output marginal :math:`h_k` is:
 #
 # .. math::
 #
-#     \widehat{h}(\vect{x}) \, = \,
-#     h(\vect{x}_0) \, + \, \sum_{i=1}^{n_{X}} \;
-#     \frac{\partial h}{\partial x_i}(\vect{x}_0).\left(x_i - x_{0,i} \right) \, +
-#     \, \frac{1}{2} \; \sum_{i,j=1}^{n_X} \;
-#     \frac{\partial^2 h}{\partial x_i \partial x_j}(\vect{x}_0).\left(x_i - x_{0,i} \right).\left(x_j - x_{0,j} \right)
+#     \widehat{h_k}(\vect{x}) \, = \,
+#     h_k(\vect{x}_0) \, + \, \sum_{i=1}^{2} \;
+#     \frac{\partial h_k}{\partial x_i}(\vect{x}_0).\left(x_i - x_{0,i} \right) \, +
+#     \, \frac{1}{2} \; \sum_{i,j=1}^{2} \;
+#     \frac{\partial^2 h_k}{\partial x_i \partial x_j}(\vect{x}_0).\left(x_i - x_{0,i} \right).\left(x_j - x_{0,j} \right)
```

Comment:
Inserting extra spaces here is an error: this is the whole purpose of the LaTeX engine.
```suggestion
#     \widehat{h_k}(\vect{x}) = h_k(\vect{x}_0) + \sum_{i=1}^{2} 
#     \frac{\partial h_k}{\partial x_i}(\vect{x}_0) \left(x_i - x_{0,i} \right) +
#     \frac{1}{2} \sum_{i,j=1}^{2} 
#     \frac{\partial^2 h_k}{\partial x_i \partial x_j}(\vect{x}_0) \left(x_i - x_{0,i} \right) \left(x_j - x_{0,j} \right)
```